classes/submission/reviewAssignment/Collector.php

```diff
@@ -284,12 +284,12 @@ public function getQueryBuilder(): Builder
         $q = DB::table($this->dao->table . ' as ra')
             // Aggregating data regarding latest review round and stage. For OMP the latest round isn't equal to the round with the highest number per submission
             ->leftJoinSub(
-                DB::table('review_assignments as rr')
+                DB::table('review_rounds as rr')
```

Comment:
Check latest round based on the review rounds table. If new review round has been started which doesn't have any review assignments, the review assignment-based approach won't show the correct result.

Reply:
Only concern here would be (if its used) that there is no where condition inside this most inner look up. So I think it calculates this for all submissions in the system.  But hard to improve it until its used in some context to see whether filtering by submissionId for example woudl make sense